content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using HPE GreenLake Console's API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Secured Yet Agile
+
+One major principle of Application Programming Interface (API) for the HPE Data Services Cloud Console (DSCC) is secured. However, the API must also be flexible for the consumption of applications or tools that use this API to extend their features using DSCC. To accommodate the flexibility, DSCC REST API is using the OAuth 2.0 authentication flow based on the client-credential which generates the limited lifetime access token. This access token will then be embedded in the header of each REST API request as the authorization bearer.
+
+This blog will walk through the essential steps required to exercise or experiment with the DSCC REST API.
+
+### Authentication Process to Obtain the Access Token
+
+The DSCC public API relies on an OAuth 2.0 third party authorization framework on behalf of the resource owner (HPE GreenLake Console's user) for security. The user starts by logging and authenticating into HPE GreenLake Console, which is authenticated by the Identity Provider (validated through username, password, or MFA). Using the API Gateway menu in GreenLake Console, customer registers their client application to obtain the OAuth 2.0 API Client Credentials (Client ID and Client Secret). This association allows user to obtain the access-token from the menu, and user can then use the access-token into any REST API call inside the token bearer field. This action allows any client application or script to perform any HTTPS method to the correct instance of DSCC.
```

Comment:
Third sentence, add "a" prior to customer so it reads:
"Using the API Gateway menu in **HPE** GreenLake Console, **a** customer registers their client application..."

Next sentence:
"This association allows **the** user to obtain the **access token** from the menu, and **the** user can then use the **access token** **with** any REST API call inside the token bearer field."

Just as you did in the platform page, you sometimes write "access token" and sometimes "access-token". It's preferred WITHOUT the hyphen. Please check through all your articles for this.